stretch-auth/src/CardContainer/CardContainer.js

```diff
@@ -5,25 +5,26 @@ import './CardContainer.css'
 
 // needs conditional renders for user & pokemon + user & no pokemon
 const CardContainer = (props) => {
+  const hasTeam = Object.values(props.userPokeTeam).some(poke => !!poke.pokemon)
```

Comment:
This line of code is great! Powerful and clean. Nice work.

Reply:
I agree, the double-bang is turning out to be super useful!